WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -330,6 +316,13 @@ on_retreat = {
 
 #Chinese traits
 son_of_heaven = {
+	potential = {
+		OR = {
+			culture_group = sinic
+			religion = confucian
+		}
+		higher_real_tier_than = KING
+	}
```

Comment:
That's fine. I'll make the Trait be used in future commitments. Until then, let this change go so that I won't have to adjust it again.